src/themes/website/styles/buttons.scss

```diff
@@ -7,3 +7,7 @@
 .btn-inline {
     padding-bottom: 1rem;
 }
+
+.btn-link {
+    text-decoration: underline;
```

Comment:
`text-decoration: underline;` - Actually, we should not do this, because users should be able to redefine link appearance.